src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -309,3 +353,32 @@ def _update_employer_metadata(
         )
 
         return pop
+
+    def calculate_income(self, idx: pd.Index) -> pd.Series:
+
+        income = pd.Series(0.0, index=idx)
+        pop = self.population_view.get(idx)
+        employed_idx = pop.index[pop["employer_id"] != data_values.UNEMPLOYED_ID]
+
+        # Get propensities for two components to get income propensity
+        personal_income_component = data_values.PERSONAL_INCOME_PROPENSITY_DISTRIBUTION.ppf(
+            pop.loc[employed_idx, "personal_income_propensity"]
+        )
+        employer_income_component = data_values.EMPLOYER_INCOME_PROPENSITY_DISTRIBUTION.ppf(
+            pop.loc[employed_idx, "employer_income_propensity"]
+        )
+        # Income propensity = probit(personal_component + employer_component)
```

Comment:
This comment is no longer correct, and I am not sure it adds anything on top of what line 372 already says.